src/main/java/com/google/api/codegen/util/ErrorMarkdownVisitor.java

```diff
@@ -0,0 +1,175 @@
+/* Copyright 2017 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.util.go;
+
+import org.commonmark.node.BlockQuote;
+import org.commonmark.node.BulletList;
+import org.commonmark.node.Code;
+import org.commonmark.node.CustomBlock;
+import org.commonmark.node.CustomNode;
+import org.commonmark.node.Document;
+import org.commonmark.node.Emphasis;
+import org.commonmark.node.FencedCodeBlock;
+import org.commonmark.node.HardLineBreak;
+import org.commonmark.node.Heading;
+import org.commonmark.node.HtmlBlock;
+import org.commonmark.node.HtmlInline;
+import org.commonmark.node.Image;
+import org.commonmark.node.IndentedCodeBlock;
+import org.commonmark.node.Link;
+import org.commonmark.node.ListItem;
+import org.commonmark.node.Node;
+import org.commonmark.node.OrderedList;
+import org.commonmark.node.Paragraph;
+import org.commonmark.node.SoftLineBreak;
+import org.commonmark.node.StrongEmphasis;
+import org.commonmark.node.Text;
+import org.commonmark.node.ThematicBreak;
+import org.commonmark.node.Visitor;
+
+/**
+ * A dummy implementation of {@code Visitor} that always throws Exception.
+ *
+ * <p>The exception is always {@code UnimplementedRenderException}. We use this instead of {@link
+ * UnsupportedOperationException} so we we don't confuse it with exceptions we get trying to add to
+ * an unmodifiable list, etc.
+ */
+public class ErrorMarkdownVisitor implements Visitor {
+
+  private static class UnimplementedRenderException extends RuntimeException {
```

Comment:
Should we fallback to copying text verbatim if we see a kind of markdown code we don't support?
Pros:
- If a markdown uses a new feature, we don't crash. We'll generate weird-looking docs, but that doesn't break users
Cons:
- Generating a lot of code, we might not look hard enough to see that some docs are now in markdown format and we never realize we need to support more Node types.